### PR TITLE
Added components for Calculator

### PR DESCRIPTION
**Pull request by devalibello**
This pull requests adds the UI of the calculator using functional components

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        <button className="button operator" type="button">+</button>
+      </div>
+
+      <div className="row row-five">
+        <button className="button button-zero" type="button">0</button>
+        <button className="button  dot" type="button">.</button>
+        <button className="button operator" type="button">=</button>
+      </div>
+    </div>
+  </div>
```
Comment (julie-ify) [inline src/components/Calculator.js]: - [ ] The calculator component should have at least one child component. Kindly add that.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+const Calculator = () => (
+  <div className="calculator">
+    <input type="text" className="input" readOnly value={0} />
+
+    <div className="buttons">
+      <div className="row-one">
+        <button className="button" type="button">AC</button>
+        <button className="button" type="button">+/-</button>
+        <button className="button" type="button">%</button>
+        <button className="button operator" type="button">/</button>
```
Comment (julie-ify) [inline src/components/Calculator.js]: - [ ] The division sign on the design is different from the one you have. Kindly use the correct sign: "&div;". You can use this: `&div;` if you do not have it. This is required for future projects.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @devalibello,

Good job so far!

## Highlights!
- Good code organization and design ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
- [ ] Kindly remove the Presentation section on the readme file if you have no presentation.

- [ ] On the readme headers, the deployment link is not working. Kindly fix that and ensure it leads to the section.

- [ ] Kindly remove the following files in the src folder: `App.test.js`, `reportWebVitals.js`, `setupTests.js`.

  image


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AnselemOdims — APPROVED** [review]
### Hi  @devalibello 🙋‍♀️, 
Good job in making the required changes 👏 

# STATUS APPROVED ✔
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Use the tag @AnselemOdims for a quicker reply


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**